packages/server/lib/routers.ts

```diff
@@ -0,0 +1,227 @@
+import bodyParser from 'body-parser';
```

Comment:
Nice, I greatly prefer having a separate file for this too. I typically call it routes which I prefer to routers but non blocking preference.

Reply:
yeah make sense, I'll rename it to routes 👌🏻